covid_xprize/examples/predictors/conditional_lstm/tests/test_conditional_xprize_predictor.py

```diff
@@ -25,12 +26,14 @@ class TestConditionalXPrizePredictor(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
-        df = load_oxford_data_trimmed(end_date=TRAINING_END_DATE, start_date=TRAINING_START_DATE)
-        df.to_csv(DATA_FILE, index=False)
+        FIXTURES_PATH.mkdir(exist_ok=True)
+        if not DATA_FILE.exists():
+            df = load_oxford_data_trimmed(end_date=TRAINING_END_DATE, start_date=TRAINING_START_DATE)
+            df.to_csv(DATA_FILE, index=False)
 
     def test_train_and_predict(self):
         predictor = ConditionalXPrizePredictor(None, DATA_FILE)
-        model = predictor.train()
+        model = predictor.train(nb_epochs=2, nb_trials=2, nb_testing_geos=2, nb_training_geos=2)
```

Comment:
Maybe you could add a comment like "Testing on a small number of epochs, trials and geos to make sure everything is wired correctly"